_schemaV2.graphql

```diff
@@ -1342,6 +1342,9 @@ type Artist implements EntityWithFilterArtworksConnectionInterface & Node & Sear
     # When true, will only return records for allowed artists.
     recordsTrusted: Boolean = false
 
+    # Show only upcoming auction results, set to false to show only past results, by default both are shown
+    showUpcoming: Boolean
```

Comment:
Interesting, this param can have three different values 🤔 Is this really required? I wonder if it could be confusing that `false` and `undefined` mean completely different things

Reply:
Actually the designs in Eigen want to include either - All (past + upcoming) or only past auction results

Reply:
All right if we specify "&upcoming=true" parameter we'll get only upcoming, otherwise all :)